src/pages/dmt5/_margin_calculator.js

```diff
@@ -0,0 +1,275 @@
+    margin-right: 2.4rem;
+    background-color: ${(props) => (props.colored ? 'var(--color-grey-25)' : 'inherit')};
+    @media ${device.tabletL} {
+        width: 100%;
+        min-height: 558px;
+    }
+`
+
+const CalculatorCard = ({ button_text, image_alt_name, image_name, link, name, text }) => {
+    const data = useStaticQuery(query)
+    return (
+        <StyledCardContainer>
+            <SubHeader align="center">{name}</SubHeader>
+            <CardText align="center">{text}</CardText>
+            <ImageWrapper>
+                <QueryImage
+                    data={data[image_name + (isBrowser() && isMobile() ? '_mobile' : '')]}
+                    alt={image_alt_name}
+                />
+            </ImageWrapper>
+            <StyledLinkButton tertiary="true" to={link} external="true" target="_blank">
+                <Localize translate_text={button_text} />
+            </StyledLinkButton>
+        </StyledCardContainer>
+    )
+}
+
+CalculatorCard.propTypes = {
+    button_text: Proptypes.string,
+    image_alt_name: Proptypes.string,
+    image_name: Proptypes.string,
+    is_mobile: Proptypes.bool,
+    link: Proptypes.string,
+    name: Proptypes.string,
+    text: Proptypes.string,
+}
+
+const calculators = [
+    {
+        index: 0,
+        name: <Localize translate_text="Margin calculator" />,
+        text: (
+            <Localize translate_text="Calculate the margin you need to open and hold your positions with our margin calculator." />
+        ),
+        image_name: 'margin_calculator',
+        button_text: <Localize translate_text="Try our margin calculator" />,
+        link: '/trader-tools#marginCalculator',
+    },
+    {
+        index: 1,
+        name: <Localize translate_text="Swap calculator" />,
+        text: (
+            <Localize translate_text="Calculate your swap fee and know exactly what you are expected to pay or will earn for maintaining an overnight contract." />
+        ),
+        image_name: 'swap_calculator',
+        button_text: <Localize translate_text="Try our swap calculator" />,
+        link: '/trader-tools/#swapCalculator',
+    },
+]
+
+const MarginCalculator = () => {
+    const settings = {
+        container_style: {
+            maxWidth: isBrowser() && isMobile() ? '100%' : '588px',
+            margin: '0',
+        },
+        slide_style: {
+            minWidth: '100%',
+            position: 'relative',
+        },
+        chevron_style: {
+            chevron_color: 'red',
+        },
+        navigation_style: {
+            nav_color: 'red',
+        },
+    }
+
+    return (
+        <SectionContainer>
+            <StyledFlexContainer>
+                <StyledFlex
+                    ai="flex-end"
+                    jc="flex-start"
+                    tablet_jc="center"
+                    fd="column"
+                    wrap="wrap"
+                    tabletL={{ mt: '0', ml: '16px', mr: '16px' }}
+                    colored={true}
```

Comment:
It's better to define these props in `StyledFlex` component

Reply:
For code reusability this left side styles uses the same StyledFlex component. These props are defined because it is specific for this left side. IMO, it is better to use a similar base style and define unique style props here